cmd/influxd/launcher/launcher.go

```diff
@@ -729,17 +744,40 @@ func (m *Launcher) run(ctx context.Context) (err error) {
 		return err
 	}
 
+	var cer tls.Certificate
+	transport := "http"
+
+	if len(m.httpTlsCert) > 0 && len(m.httpTlsKey) > 0 {
```

Comment:
Typically for empty string values people just check equality with `""` in go since all strings are constant.

Reply:
@desa Updated as instructed